.github/scripts/python-pkg-install.sh

```diff
@@ -0,0 +1,6 @@
+#/bin/bash
+
+pip3 install -e python_apps/airtime_analyzer/.
+pip3 install -e python_apps/airtime-celery/.
+pip3 install -e python_apps/api_clients/.
+pip3 install -e python_apps/pypo/.
```

Comment:
are we gonna add more `python_apps`? Maybe it make sense to automate installing them on GH?
```suggestion
for app in `ls python_apps`; do
  pip3 install -e python_apps/$app
done
```

Reply:
Automating sounds good to me. I was just copying over what we had so far. With only four apps, it's just one more line of code to write. 🤷

Reply:
Moved to for list